src/cephadm/cephadm

```diff
@@ -1656,6 +1656,8 @@ def infer_config(func):
                 config = '/var/lib/ceph/{}/{}/config'.format(ctx.fsid,
                                                              name)
         if config:
+            if not os.path.isfile(config):
+                raise Error('Cannot infer a config')
```

Comment:
This error message is hard to understand. Can we raise `FileNotFoundError` instead and show the full path instead?

Reply:
Why don't we catch `FileNotFoundError`?

Reply:
your right this check is not really needed. the try execpt in get_config_and_keyring will catch this.